src/snowflake/telemetry/_internal/opentelemetry/exporter/otlp/proto/common/log_encoder.py

```diff
@@ -0,0 +1,22 @@
+from logging import LogRecord
+
+from typing import Sequence
+
+from opentelemetry.proto.collector.logs.v1.logs_service_pb2 import ExportLogsServiceRequest
+from opentelemetry.proto.logs.v1.logs_pb2 import LogsData
+
+
+def encode_logs(batch: Sequence[LogRecord]) -> ExportLogsServiceRequest:
```

Comment:
Do we need this at all? Can we just call directly into `serialize_logs_data`?

Reply:
This is a non-public method that matches the same pattern that we used in metrics_encoder and trace_encoder. Since it's not part of the API, we can modify this implementation detail once we have our own logging handler that uses native Python LogRecords (instead of the OpenTelemetry version of LogRecords).